File: pset6/mario/less/mario.py
# built the pyramid of mario using hashes

from cs50 import get_int

# getting user input but valid
while True:
    height = get_int("Height: ")
    if (height > 0 and height < 9):
        break
# build a pyramid
for i in range(height):
    print((height - (i + 1)) * " ", end="")
    print((i + 1) * "#")

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Getting User input ;
    int input;
    do
    {
        input = get_int("Height: ");
    }
    while (input > 8 || input < 1);

    //Logic for building hashes;
    for (int i = 0; i < input; i++)
    {
        //printing spaces
        for (int s = input - 1; s > i; s--)
        {
            printf(" ");
        }

        for (int n = 0; n < i; n++)
        {
            printf("#");
        }
        printf("#\n");
    }










}